services/database/relational/taxonomy.md

```diff
@@ -20,3 +20,4 @@ use in financial services ecosystems.
 | CCC-RDMS-10  | Logging | Configurable logs are available for user inspection. |
 | CCC-RDMS-11 | Monitoring | Configurable metrics are available for user inspection. |
 | CCC-RDMS-12 | Alerting | Configurable alerts can be enabled. |
+| CCC-RDMS-13 | Failover | Standby database is trasitioned to the primary after the primary fails or become unreachable. |
```

Comment:
```suggestion
| CCC-RDMS-13 | Failover | Standby database can be implemented for failover when the primary can't be reached. |
```

Reply:
consider rephrasing this to make it clear that this feature should be made available to users, rather than always automatically deployed (that would be part of the hardening controls)